arlo_server/audit_boards.py

```diff
@@ -251,11 +251,15 @@ def is_round_complete(election: Election, round: Round) -> bool:
         .having(func.count(AuditBoard.id) == 0)
         .count()
     )
-    all_audit_boards_set_up = num_jurisdictions_without_audit_boards_set_up == 0
-    all_audit_boards_signed_off = all(
-        ab.signed_off_at is not None for ab in round.audit_boards
+    num_audit_boards_with_ballots_not_signed_off: int = (
+        AuditBoard.query.filter_by(round_id=round.id, signed_off_at=None)
+        .join(SampledBallot)
```

Comment:
is this going to yield the right number of audit boards? I think it yields the number of *ballots* across those audit boards, unless SQLAlchemy does something unexpected.

Reply:
Good q. I thought about this one but got lazy. You're right that it will yield the number of ballots.

I'm pretty sure since we're just checking whether or not the count is zero, it's logically equivalent to ask:
- Are there any audit boards with at least one ballot assigned that didn't sign off?
- Are there any ballots assigned to an audit board that didn't sign off?

But it's confusing so I'll just be more precise with the query :)

Reply:
👍